### PR TITLE
feat(query): added Autoscaling Groups Supply Tags for Terraform

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- added a query to check if Autoscaling Groups supply tags to launch configurations

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Comment by rogeriopeixotocx** [comment]
@rjegoncalves @JoaoDuarteGomes please approve the category and severity

File: assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ba48df05-eaa1-4d64-905e-4a4b051e7587",
+  "queryName": "Autoscaling Groups Supply Tags",
+  "severity": "LOW",
+  "category": "Encryption",
```
Comment (JoaoDuarteGomes) [inline assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json:5]: This query does not fit this category. This should be in the "Availability" category.

File: assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "descriptionText": "Autoscaling groups should supply tags to configurate",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group#launch_configuration",
```
Comment (JoaoDuarteGomes) [inline assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json:7]: Please review this link to ensure it is correct.

**Review by JoaoDuarteGomes — CHANGES_REQUESTED** [review]
The category needs to be reviewed.

File: assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json (hunk 2 above)
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json:7]:
```suggestion
  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group#tag-and-tags",
```

File: assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json (hunk 1 above)
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json:5]:
```suggestion
  "category": "Availability",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by JoaoDuarteGomes — APPROVED** [review]
LGTM.

**Review by rogeriopeixotocx — APPROVED** [review]

**Pull request merged**